recipes/libtins/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+from conans import tools, CMake, ConanFile
+import os
+
+
+class LibTinsConan(ConanFile):
+    name = "libtins"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/mfontanini/libtins"
+    description = "High-level, multiplatform C++ network packet sniffing and crafting library."
+    license = "BSD-2-Clause"
+    topics = ("pcap", "packets", "network", "packet-analyser", "packet-parsing", "libpcap", "sniffing")
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "cmake_find_package", "cmake_find_package_multi"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_cxx11": [True, False],
+        "with_ack_tracker": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_cxx11": True,
+        "with_ack_tracker": True
+    }
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("libpcap/1.10.0")
+        self.requires("openssl/1.1.1i")
+        if self.options.with_ack_tracker:
+            self.requires("boost/1.75.0")
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.options.with_cxx11:
+            tools.check_min_cppstd(self, 11)
```

Comment:
cppstd might be not set
You should add if to check if it is set or not

Reply:
It looks like this check is already inside tools.